packages/akashic-cli-serve/src/client/store/DevtoolUiStore.ts

```diff
@@ -45,4 +47,9 @@ export class DevtoolUiStore {
 		this.eventEditContent = content;
 		storage.put({ eventEditContent: content });
 	}
+
+	@action
+	updateEntityList(entityList: string): void {
```

Comment:
エンティティツリーを直接編集することや外部JSONファイルの内容に書き換えみたいなことはsandboxではできないはずなのでこのメソッドは不要な気がしました。(sandboxにそのような機能があるもしくはserveに新たに追加するということであればすみません)

Reply:
直接 `devtoolUiStore#entityList` を更新する、という意図でしょうか。

actionを経由せずにobservableを更新すると、下記のエラーが発生するため、actionを通しています。
`Error: [mobx] Since strict-mode is enabled, changing observed observable values outside actions is not allowed. Please wrap the code in an `action` if this change is intended. Tried to modify: DevtoolUiStore@4.entityList`

Reply:
`PlayOperator#udpateEntityList`があるからこのメソッドは要らないと思って書いてしまいましたが、そこで使われてましたね。完全に勘違いでした、、すみません。